packages/parrot-devtools/src/app/components/Middleware.jsx

```diff
@@ -17,15 +17,20 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
 class Middleware extends Component {
+  // eslint-disable-next-line react/state-in-constructor
   state = {
     loading: true,
   };
 
+  // eslint-disable-next-line react/no-deprecated
   componentWillMount() {
```

Comment:
Since we are on latest React version. Let's use `UNSAFE_` prefix for now. Even better if you can refactor this without using any of the soon-to-be-deprecated lifecycle methods. Maybe give it a shot in this PR and see if the tests pass.

Reply:
I refactored the components to use hooks in 824f96d instead of the prefix. I think I got everything in place...